package.json

```diff
@@ -66,6 +82,16 @@
                 {
                     "command": "aks.periscope",
                     "when": "view == kubernetes.cloudExplorer && viewItem =~ /aks\\.cluster/i"
+                },
+                {
+                    "command": "azure-deploy.BrowseCICDPipeline",
```

Comment:
💡 With camel casing changes this and the one below will be effected as well. (Changes for consistency)

Reply:
:bulb: I think for consistent camel casing like “selectSubscripts” I.e. I was referring to lower camel casing, so say for example: “BrowseCICDPipeline” will become “browseCICDPipeline or browserCicdPipeline”

This will impact the other resolves camel casing comments to be consistent with the AkS extension commands casing.

Just setting this one as unresolved but it stands true for both the commands. Thank you.